### DISCUSSION TITLE
Compatibility with UXBOX?

### DISCUSSION THREAD

**Discussion by 12people** [akiraux/Akira#471] [Ideas]
[UXBOX](https://github.com/uxbox/uxbox) is an up-and-coming FOSS online prototyping tool with similar goals to Akira.

It'd be great if UXBOX and Akira were cross-compatible, allowing one to edit UXBOX files with Akira and vice versa. Ideally, there'd be one file format behind both pieces of software.

Has there been any communication between the two projects?

**Comment by albfan** [comment]
Seems a different paradigm. They use a backed/frontend.

If you find the specs for UXBOX, we are open to create/accept export/import implementation.

Surely is not now in the priority list, but I'm sure our format version for Akira files is just an abstraction of read/write whatever.

**Comment by 12people** [comment]
@albfan Yes, it's a web application, but the concept is the same as Akira's: an open-source vector editor for UX designers. The only difference is the technologies – Akira is to UXBOX as Sketch is to Figma.

The UXBOX team gave [a talk on their software](https://invidio.us/watch?v=2_vIpNtWu6Q) at FOSDEM recently. They hope to base their editor on SVG. Given that Akira files are also based around SVG (if I'm not mistaken), it seems to me there is potential for establishing a common format between the two.

It'd be great to have a standard format that all the open-source UX-centric vector editors agree on and support.

**Comment by Alecaddd** [comment]
Thank you so much for raising this up.
I'll definitely get in contact with them once we have a stable beta and a fully fledged out file system.
I'm totally open to collaborate and trying to define an open standard which can allow applications to collaborate.

**Comment by 12people** [comment]
Awesome, great to hear!
